src/Module/Configuration/Form/CarriersForm.php

```diff
@@ -1262,6 +1262,41 @@ private function getExtraTabFields(Carrier $carrier, string $prefix = ''): array
                 'name'  => $prefix . Constant::INSURANCE_CONFIGURATION_NAME,
                 'desc'  => $this->module->l('Package will be insured according to below settings when Always insure package is on, or any product in the order has insurance set to on.', 'carriers'),
             ];
+            try {
+                $c = ConsignmentFactory::createByCarrierId($myParcelCarrier->getId());
+                $c->setPackageType(AbstractConsignment::PACKAGE_TYPE_PACKAGE);
+                $insurancePossibilities = array_merge([0], $c->getInsurancePossibilities());
```

Comment:
Misschien is het mooi om die `[0]` in een const zetten als `DEFAULT_INSURANCE_POSSIBILITIES` oid